### PR TITLE
Maddie data hook upto list compo

### PR DESCRIPTION
**Pull request by Prtfw**
Note: apikey / query string doesn't work because it's on local host and the api key is for the production domain.
But other than that we have a full cycle going from api and back.
if you see > 4 results on the page then you know it's real api data, if you see 4, that's the fake api data. 
Left some console logs for future debugging. (mostly removed)

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
line 21 in src/App.js should read 
"this.state = initialState"
initialState is spelled incorrectly

File: src/App.container.js [hunk 1]
```diff
@@ -1,20 +1,37 @@
...
+// const actions = {
+//   onClickShoppingCartLogo: openWishlistPanel,
+//   onClickClosePanel: closeWishlistPanel,
+//   updateSearchText,
+//   getResults: getAPIResults, // !!! import from action properly
+// }
+
+
+
+// export default connect(mapStateToProps, actions)(App);
```
Comment (astr0-4) [inline src/App.container.js]: I think you can remove this commented out stuff

File: src/epics/index.js [hunk 2]
```diff
@@ -1,12 +1,235 @@
...
+import { combineEpics } from "redux-observable";
+import { GET_API_RESULTS, sendAPIResults } from "../redux/actions";
 
 
-//epics go here
-const testEpic = action$ =>
-  action$.ofType('')
-  .mergeMap(action => action.payload)
-  .map(response => console.log(response));
+const LCBO_access_key = 'MDo4OTc1NDY4Ni03MDAzLTExZTctOTNkNS04ZmJjMjZkMWQ2NTE6cHFlN3BwUEtzaDJ6aUpidHU4QnZTOU1RODVUSFZVd0RhRUc5'
```
Comment (astr0-4) [inline src/epics/index.js]: I think the convention for constants like this is all uppercase

File: src/epics/index.js [hunk 3]
```diff
@@ -1,12 +1,235 @@
...
+  action$
+    .ofType(GET_API_RESULTS)
+    .mergeMap(action => fetchProducts(action.payload)) //TODO: use search string later.
+    .map(res => {
+      return sendAPIResults(res)});
+
+export default combineEpics(apiFetchEpic);
+
+
+var sampleData = [
```
Comment (astr0-4) [inline src/epics/index.js]: I think it would be better to put this in a separate file.

**Review by ghost — COMMENTED** [review]
disregard comment about line 21...the variable is spelled the same in both line 11 and line 21. You can either leave as is or revise the spelling in both lines.

File: src/epics/index.js [hunk 4]
```diff
@@ -1,12 +1,235 @@
...
+
+const apiFetchEpic = action$ =>
+  action$
+    .ofType(GET_API_RESULTS)
+    .mergeMap(action => fetchProducts(action.payload)) //TODO: use search string later.
+    .map(res => {
+      return sendAPIResults(res)});
+
+export default combineEpics(apiFetchEpic);
+
```
Comment (astr0-4) [inline src/epics/index.js]: woooooooo

**Review by ghost — APPROVED** [review]

File: src/redux/actions/header.actions.js [hunk 5]
```diff
@@ -8,7 +8,7 @@ export const updateSearchText = searchText => ({
...
 })
 
-export const getAlcoholList = alcohol => ({
-	type: HEADER_ACTIONS.GET_ALCOHOL_LIST,
-	payload: alcohol,
-})
+// export const getAlcoholList = alcohol => ({
+// 	type: HEADER_ACTIONS.GET_ALCOHOL_LIST,
+// 	payload: alcohol,
+// })
```
Comment (astr0-4) [inline src/redux/actions/header.actions.js]: do we still need this?

**Review by astr0-4 — CHANGES_REQUESTED** [review]
wooo! It's awesome to see some live data in here! 🎉  🎉 

Just a couple small things and this should be good to go.

File: src/App.js [hunk 6]
```diff
@@ -1,24 +1,40 @@
...
+
+
+import './App.css';
+import 'rxjs';
+
+import Header from './components/Header';
+import ProductList from './components/ProductList';
+import WishlistPanel from './components/WishlistPanel';
+
+const intialState = {
```
Comment (astr0-4) [inline src/App.js]: this should be initial state

Comment (astr0-4) [inline src/App.js]: should be initialState

**Review by astr0-4 — CHANGES_REQUESTED** [review]

File: package.json [hunk 7]
```diff
@@ -3,8 +3,10 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "axios": "^0.16.2",
     "express": "^4.15.3",
     "helmet": "^3.6.1",
+    "jQuery": "^1.7.4",
```
Comment (pkanal) [inline package.json]: If we're not using jquery - lets remove it from package.json

**Comment by Prtfw** [comment]
@pkanal @astr0-4 @holdersimone Changes made. Thanks for the cod review + comments. Eih... I see my dyslexia kicked in : ) but at least I'm consistent in incorrect spellings.

**Comment by astr0-4** [comment]
@Prtfw LGTM!

**Review by astr0-4 — APPROVED** [review]

**Pull request merged**